pkg/sdn/plugin/controller.go

```diff
@@ -339,7 +340,12 @@ func (plugin *OsdnNode) SetupSDN(localSubnetCIDR, clusterNetworkCIDR, servicesNe
 func (plugin *OsdnNode) SetupEgressNetworkPolicy() error {
 	policies, err := plugin.registry.GetEgressNetworkPolicies()
 	if err != nil {
-		return fmt.Errorf("Could not get EgressNetworkPolicies: %s", err)
+		if kapierrs.IsForbidden(err) {
```

Comment:
There's a bunch of work happening here, but short answer is we can't
do this safely yet.  In the long term a taint would be excellent.